lib/jekyll-import/importers/wordpressdotcom.rb

```diff
@@ -22,16 +22,21 @@ def self.specify_options(c)
       end
 
       # Will modify post DOM tree
-      def self.download_images(title, post_hpricot, assets_folder)
+      def self.download_images(title, post_hpricot, assets_folder, published_at)
```

Comment:
This changes the signature of a public method. Will you be able to pass the `published_at` attribute without changing the arity (no. of arguments a method is called with)?

Reply:
The call to this method on line 199 has been updated with the extra parameter
Unless there is another place I'm not aware of?

Reply:
Restored the method signature.  moved the logic earlier and guarded it in case there are pages or posts without a publish date